runnc-cli/state.go

```diff
@@ -1,9 +1,11 @@
 package main
 
 import (
-	"fmt"
-
+	"encoding/json"
+	"github.com/opencontainers/runc/libcontainer/utils"
```

Comment:
Good point. Right now, the main reason for re-implementing the libcontainer code is that we want to define our own `configs`, since we do not need a lot of additional steps `runc` does.

I will add the copyright at the moment, and look in the future whether we can abstract the libcontainer library itself sometime later down the road. I think we can probably factor out abstract of the libcontainer code, but the code in `runnc-cli` will probably need to be rewritten directly.